.github/workflows/ci.yml

```
@@ -0,0 +1,33 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on: [ push, pull_request, pull_request_review, pull_request_review_comment ]
```

Do `pull_request_review` and `pull_request_review_comment` make sense here? PR reviews and comments don't change anything in the code, so the result of our tests will remain the same (assuming our tests are deterministic which they should be).

> **Note:** Suggestions in reviews or comments need to be manually accepted before they are part of the PR, so even they don't change the code by themselves.

```suggestion
on: [ push, pull_request ]
```